test/e2e/create_test.go

```diff
@@ -304,6 +312,20 @@ func (tc *testContext) setPowerShellDefaultShell(machine *mapi.Machine) error {
 	})
 }
 
+// disableWindowsDefender uninstalls Windows Defender from the given machine. Reboots the underlying VM.
```

Comment:
You are uninstalling the feature and naming the function disable which is misleading. So how about `uninstallWindowsDefender`?